lib/omnibus/fetchers/git_fetcher.rb

```diff
@@ -203,7 +203,7 @@ def contains_revision?(rev)
     #   the shellout object
     #
     def git(command)
-      shellout!("git #{command}", cwd: project_dir)
+      shellout!("git -c core.autocrlf=false #{command}", cwd: project_dir)
```

Comment:
AFAIK the crlf stuff only really applies to Windows, and for the kind of software we're building and the way we build it, we want it off everywhere.